sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/handler/ConnectionHandler.java

```diff
@@ -113,6 +113,11 @@ public int getMaxFrameSize() {
      * @param transport Transport to add layers to.
      */
     protected void addTransportLayers(Event event, TransportInternal transport) {
+        // default connection idle timeout is 0.
+        // Giving it a idle timeout will enable the client side to know broken connection faster.
+        // Refer to http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-transport-v1.0-os.html#doc-doc-idle-time-out
+        transport.setIdleTimeout(60_000);
```

Comment:
Can you set this to operationTimeout in ms? So it's configurable in some way. or creating a constant that can be referenced.

Reply:
Users might set a short operationTimeout, which as an adverse impact on the established connection. So a separate value might be better.
Added constant CONNECTION_IDLE_TIMEOUT